public/css/stylesheet.css

```diff
@@ -0,0 +1,275 @@
+/* - - - - -  Landing Page CSS - - - - - */
```

Comment:
I had that before and Jared said to put it all into one big file....?
I was also given the feedback to use webpack to minify/bundle everything. Just haven't incorporated the feedback.